kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkManager.java

```diff
@@ -40,16 +36,19 @@
   // TODO: We want to move this to the config eventually
   // Less than KaldbDistributedQueryService#READ_TIMEOUT_MS
   public static final int QUERY_TIMEOUT_SECONDS = 10;
-  public static final int LOCAL_QUERY_THREAD_POOL_SIZE = 4;
```

Comment:
Let's leave the static variables in place and them `MIN_QUERY_THREAD_POOL_SIZE`  and `MAX_QUERY_THREAD_POOL_SIZE`.